PatientRegistrationApp/src/components/SinglePatientPersonalInfo.vue

```diff
@@ -218,6 +219,19 @@ export default {
         valid = false;
       }
 
+      if( this.patientPhoto ) {
+        if (this.patientPhoto.size > 2097152) {
+          if (!valid) {
+            message += "\n";
```

Comment:
Could we not combine these two messages? I also feel like there's some better logic to handle this if-else block. It's basically the same code but only differing on the `\n`...

Would this work? (really top the head haven't thought it through...)
`if(!valid){ message += "\n";}
message += "...";`
Just get rid of the else since we only need to add the `\n` in if valid isn't true but the photo comment is always in?

Reply:
I hide the message "Woops! You need to enter the following field(s):" when there's no other invalidation's because the message makes no sense for the image requirements.

Reply:
I did change the logic here a little.